src/IO/GEHDF5Wrapper.cxx

```diff
@@ -303,22 +339,22 @@ Succeeded GEHDF5Wrapper::initialise_singles_data(const std::string &path)
     return Succeeded::yes;
 }
 
-Succeeded GEHDF5Wrapper::initialise_proj_data_data(const std::string& path,
+Succeeded GEHDF5Wrapper::initialise_proj_data(const std::string& path,
                                                  const unsigned int view_num)
 {
     if(path.size() == 0)
     {
-        if(is_signa)
+        if(rdf_ver==9) 
         {
             m_address = "/SegmentData/Segment2/3D_TOF_Sinogram/view";
+            // AB: todo norm3d, geo3d, rdf.0 do not have this info. Only rdf.0 has "Segment2/3D_TOF_Sinogram", but no "view" in sight. 
```

Comment:
definitely only appropriate for sinos.

Possibly you have a compressed sino if there's no `view`

Reply:
@KrisThielemans hum possibly.... I have `cvt` and then `pwbX` x=[1,15], 16 Datasets in total. Trying to figure out how to know if its compressed or not.

Reply:
checking some Matlab code, it seems that
`HeaderData/Sorter/Segment2` etc should have a field `compDataSegSize`. If it's zero, it is not compressed.

Reply:
```
 h5read([path 'rdf.0'], '/HeaderData/Sorter/Segment2/compDataSegSize')

ans =
   14849736
```
Hum seems compressed then.